pymatgen/command_line/bader_caller.py

```diff
@@ -106,7 +106,8 @@ class BaderAnalysis:
               " Please download the library at http://theory.cm.utexas"
               ".edu/vasp/bader/ and compile the executable.")
     def __init__(self, chgcar_filename, potcar_filename=None,
-                 chgref_filename=None, parse_atomic_densities=False):
+                 chgref_filename=None, parse_atomic_densities=False,
+                 vacuum_mode="auto"):
```

Comment:
Can we make this an enum to provide some unambiguity? I think in general we should do that for anything that is a selection of choices.